articles/quickstart/webapp/rails/01-login.md

```diff
@@ -273,8 +191,8 @@ In `logout_helper.rb` file add the methods to generate the logout URL.
 
 module LogoutHelper
   def logout_url
-    domain = Rails.application.secrets.auth0_domain
-    client_id = Rails.application.secrets.auth0_client_id
+    domain = Rails.application.config.auth0.auth0_domain
+    client_id = Rails.application.config.auth0.auth0_client_id
```

Comment:
Above, the following syntax was used `Rails.application.config.auth0['auth0_client_id'],`, should we stick to one syntax or is there any reason these use a different way to access the properties?

Reply:
I think we should use `auth0_config = Rails.application.config_for(:auth0)` here again and index into that instead of `Rails.application.config.auth0`